core/src/main/java/com/nerds/gamejam/ecs/system/CameraSystem.java

```diff
@@ -67,6 +77,15 @@ protected void processSystem() {
         } else if (InputUtil.isKeyPressed(Input.Keys.E)) {
             this.camera.zoom -= 1f * world.getDelta();
         }
+
+        if (Gdx.input.isKeyJustPressed(Input.Keys.SPACE)) {
+
+        }
+
+        if (InputUtil.isKeyPressed(Input.Keys.ESCAPE)) {
```

Comment:
Probably best moved to GameJam as its not camera related and do something like

	@Override
	public void render() {
		super.render();
               if (InputUtil.isKeyPressed(Input.Keys.ESCAPE)) {
               ...
	}

Reply:
I'll just blat this, I added it because it was PITA to close when it was fullscreen